WooCommerce/WooCommerceTests/Authentication/NotWPAccountViewModelTests.swift

```diff
@@ -145,3 +277,106 @@ private extension NotWPAccountViewModel {
         self.init(error: SignInError.invalidWPComEmail(source: .wpCom))
     }
 }
+
+
+// MARK: - WordPressAuthenticator Initialization
+
+private extension NotWPAccountViewModelTests {
```

Comment:
Thanks for adding this! I got similar failures when running tests for `ULErrorViewController` and `ULAccountMismatchViewController` locally as well - but since the tests don't fail on CI, I've been ignoring that 😅 Do you think we should make this method an extension for `WordPressAuthenticator` accessible in all the unit test target? This can be useful for running the other tests above.

Reply:
I got this from `ULErrorViewControllerTests` file. I have moved this to an internal file inside `WooCommerceTests` target. 
Done in https://github.com/woocommerce/woocommerce-ios/pull/7954/commits/24616cb2fb57d43e1d286eba3ea3ad4bb56f4e44